source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -251,6 +251,7 @@ void BioXASAppController::setupUserInterface()
 	addComponentView(BioXASBeamline::bioXAS()->standardsWheel(), "Standards Wheel");
 	addComponentView(BioXASBeamline::bioXAS()->cryostatStage(), "Cryostat Stage");
 	addComponentView(BioXASBeamline::bioXAS()->filterFlipper(), "Filter Flipper");
+	addComponentView(BioXASBeamline::bioXAS()->detectorStageLateral(), "Ge 32-el Stage");
```

Comment:
Good eye, @XuetaoLiu-CLS! By default, the getters defined in BioXASBeamline always return 0, unless it's for a component that's created in that class. For Main, this call would return 0. This condition is checked in createComponentView().